app/models/measurement.rb

```diff
@@ -10,11 +9,8 @@ class Measurement < ActiveRecord::Base
 
   #input: measurement
   def bmi_calc
-    #first = last, because of ordering in model
-    weight = self.weight
-    height = self.person.length
+    height = person.length
```

Comment:
There are two things here:

 1. The `height` variable, which contains `person.length` is only used once. `person.length` is descriptive enough to just use it directly, without storing it in a local variable `height` first. There is no performance penalty here, just readability.
 2. IMHO you should store this calculated value in the database as you measure weight once, and that is it - once measure it does not change. There is a performance gain here when displaying lots of BMI values at once, like in an overview or when drawing a BMI graph or something. As I understand, this was done as an exercise, so it's okay to have it like this.